includes/wikia/transaction/TransactionTraceNewrelic.php

```diff
@@ -38,7 +38,8 @@ function __construct() {
 
 		// report request ID as a custom request parameter for easier debugging of slow transactions
 		if ( function_exists( 'newrelic_add_custom_parameter' ) ) {
-			newrelic_add_custom_parameter( 'requestId', RequestId::instance()->getRequestId() );
+			newrelic_add_custom_parameter( 'requestId', WikiaTracer::instance()->getTraceId() );
```

Comment:
I don't think anything is "automatically" consuming this.  So if the field changes everywhere, it should be fine to change it here too.